client/pages/_app.tsx

```diff
@@ -1,62 +1,62 @@
-import React from "react";
-import "../styles/globals.css";
-import type { AppProps } from "next/app";
+import React from 'react'
+import '../styles/globals.css'
+import type { AppProps } from 'next/app'
 
 interface AppData {
-  clubs: Map<ClubId, ClubInfo>;
+  clubs: Map<ClubId, ClubInfo>
```

Comment:
we're going no semi-colons :0 should we change the frontend for consistency?

Reply:
Don't worry about changing it manually; if we care enough to standardize formatting then we'll find a way to automate it!

(prettier)